code-experiments/src/coco_problem.c

```diff
@@ -523,6 +523,15 @@ const double *coco_problem_get_largest_values_of_interest(const coco_problem_t *
   return problem->largest_values_of_interest;
 }
 
+const double *coco_problem_get_largest_fvalues_of_interest(const coco_problem_t *problem) {
```

Comment:
Shouldn't the returned nadir point be a vector of length #objectives? Why do we return a double only?

Reply:
We return `const double *`

Reply:
Ah, sorry. Didn't see it. You are right.